### PR TITLE
Hide (or keep, like status quo) header and footer on pagebreak-ed empty pages

### PR DESCRIPTION
**Pull request by Leedehai**
Fixing #2722.

### CODE REVIEW

**Comment by Leedehai** [comment]
Yet, if maintainers think it's valuable to provide an option so users can choose whether they want to hide the marginals (headers, footers) then I can make the changes as well. I guess that would introduce a new argument to the `pagebreak()` function called `keep-marginals` that defaults to `false`, but IMHO that complication is unnecessary because there _seems_ to be no real use cases for `keep-marginals: true`.

**Comment by Leedehai** [comment]
The updated commit supports a new option `keep-marginals`, as suggested in #2722.

File: tests/typ/layout/pagebreak-parity.typ [hunk 1]
```diff
@@ -33,3 +33,28 @@ Second
...
+  ]
+)
+
+First
+#pagebreak(to: "odd")
+Third
+
+---
+// Test headers and footers are skipped on pagebreak-ed empty pages,
+// but with keep-marginals = true
```
Comment (kwfn) [inline tests/typ/layout/pagebreak-parity.typ:51]:
```suggestion
// but with `keep-marginals: true`.
```

Comment (Leedehai) [reply tests/typ/layout/pagebreak-parity.typ]: Thanks! Though I decided to combine the tests..

**Comment by laurmaedje** [comment]
I've discussed this with @reknih and we've come to the conclusion that this is not a good argument to have on `pagebreak`. The requirement to not have headers on empty pages exists, but is not specific to `pagebreak(to: "even" | "odd")`.

I think we need to go back to the design board on this. I'll post some thoughts in #2722.

**Pull request closed without merging**